README.md

```diff
@@ -330,6 +331,28 @@ need to reprocess all older data to handle taxonomy changes and to keep
 everything consistent.  Also keep this in sync with the version of the taxonomy
 in prepare-dashboard-data.sh.
 
+### BowTie2
+
+#### Install
+
+```
+wget -O bowtie2.zip "https://sourceforge.net/projects/bowtie-bio/files/bowtie2/2.5.2/bowtie2-2.5.2-linux-x86_64.zip/download" 
+unzip bowtie2.zip 
+rm bowtie2.zip
+```
+
+#### Set up Bowtie2 database
+
+To create a Bowtie2 database, we need to download genomes from NCBI, using ncbi-genome-download. To run the affiliated script `gimme_taxa.py`, you will also need to install the dependencies `ete3` and `six`. 
+
+```
+pip install ncbi-genome-download
+python -m pip install ete3 six
```

Comment:
`python -m pip install ete3 six ncbi-genome-download`

(`python -m pip install` and `pip install` do the same thing, except `python -m pip` handles cases where your `pip` and your `python` disagree about which site-packages dir to use.)